src/http_request.rs

```diff
@@ -225,8 +226,9 @@ async fn http_post(
 #[derive(Clone)]
 #[wasm_bindgen(getter_with_clone)]
 pub struct InitTunnelResult {
-    pub(crate) client: ntor::client::NTorClient,
-    pub(crate) ntor_session_id: String,
+    pub(crate) client: NTorClient,
+    pub(crate) token1: String,
```

Comment:
Here's the culprit.... there has got to be an alias to change the variable name after deserialization no?